### PR TITLE
Branch 3.1 Register Apptenitve lifecycle callbacks without modifying the Application class

### PR DESCRIPTION
**Pull request by barry-li**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ v3.0.0 - May 02, 2016
```
Comment (barry-li) [inline CHANGELOG.md]: This is the merge from 3.0.0 release

File: src/android/ApptentiveBridge.java [hunk 2]
```diff
@@ -58,13 +59,24 @@
...
+                currentActivity.runOnUiThread(new Runnable() {
+                    @Override
+                    public void run() {
+                        Apptentive.register(currentActivity.getApplication());
+                        ApptentiveInternal.getInstance().getRegisteredLifecycleCallbacks().onActivityCreated(currentActivity, null);
+                        ApptentiveInternal.getInstance().getRegisteredLifecycleCallbacks().onActivityStarted(currentActivity);
+                        ApptentiveInternal.getInstance().getRegisteredLifecycleCallbacks().onActivityResumed(currentActivity);
+                    }
+                });
+            }
```
Comment (skykelsey) [inline src/android/ApptentiveBridge.java]: Is it ever possible for this to run more than once?

File: www/ApptentiveAndroid.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 var Apptentive = {
-
+
```
Comment (skykelsey) [inline www/ApptentiveAndroid.js]: Whitespace only change. Let's back this out.

File: src/android/ApptentiveBridge.java (hunk 2 above)
Comment (skykelsey) [inline src/android/ApptentiveBridge.java]: What if a customer already extended the Application object? We should handle that case right? Could it be as simple as checking to see if registration has already occurred?

**Comment by skykelsey** [comment]
:+1:

**Pull request merged**